.vscode/settings.json

```diff
@@ -5,7 +5,6 @@
     "flow.useNPMPackagedFlow": true,
     "flow.enabled": true,
     "javascript.validate.enable": false,
-    "prettier.eslintIntegration": true,
```

Comment:
My editor was complaining about this setting. I can remove if we prefer, but vscode doesn't like this anymore.

Reply:
yeah it may have been removed?